sechub-pds-solutions/iac/docker/IaC-Debian.dockerfile

```
@@ -22,21 +23,22 @@ FROM ${BASE_IMAGE} AS builder
 # Build args
 ARG GO
 ARG PDS_ARTIFACT_FOLDER
-
 ARG BUILD_FOLDER="/build"
-ARG GIT_URL_KICS="https://github.com/Checkmarx/kics.git"
-ARG GIT_BRANCH_KICS="master"
+ARG KICS_VERSION="2.0.1"
```

KICS_VERSION should be inherited from the base container (line 9)
```suggestion
ARG KICS_VERSION
```